app/page.tsx

```diff
@@ -144,15 +183,35 @@ export default function HomePage() {
   const showPaginationButtons = hasMoreMarkets || page !== 1;
 
   return (
-    <div className="justify-center px-6 mt-12 space-y-12 md:px-10 lg:px-20 xl:px-40 md:flex md:flex-col md:items-center">
+    <div className="justify-center px-6 mt-12 space-y-8 md:px-10 lg:px-20 xl:px-40 md:flex md:flex-col md:items-center">
+      <div className="w-full">
+        <h1 className="text-2xl font-semibold text-white capitalize">
+          🔮 {category ? category : "All"}
+        </h1>
+      </div>
```

Comment:
Maybe we can remove that extra `<div>`

```suggestion
      <h1 className="text-2xl font-semibold text-white w-full capitalize">
        🔮 {category ? category : "All"}
      </h1>
```

Reply:
Yes, good catch.